src/utils/components/CoordinateCopySlot.vue

```diff
@@ -70,9 +77,22 @@ function setTooltipContent() {
         copyTooltip.setContent(i18n.t('copy_cta'))
     }
 }
+
+function display(coordinates) {
+    let displayedFormat
+    if (coordinateFormat.value === 'None') {
+        return coordinates
+    }
+    if (coordinateFormat.value) {
+        displayedFormat = allFormats.find((format) => format.id == coordinateFormat.value)
+    } else {
+        displayedFormat = allFormats.find((format) => format.id === displayedFormatId.value)
+    }
+    return displayedFormat.format(coordinates, projection.value, true)
+}
```

Comment:
Based on having `coordinateFormat` being of type `CoordinateFormat` you can simplify this function

```javascript
function display(coordinates) {
    if (coordinateFormat.value) {
        return coordinateFormat.value.format(coordinates, projection.value, true)
    }
    return coordinates
}
```

The format should be given in the FeatureStyleEdit.vue and FeatureDetail.vue